common/config.h

```diff
@@ -82,6 +83,7 @@
 // repo defaults
 #define TDNF_DEFAULT_REPO_LOCATION        "/etc/yum.repos.d"
 #define TDNF_DEFAULT_CACHE_LOCATION       "/var/cache/tdnf"
+#define TDNF_DEFAULT_VARS_DIRS            "/etc/tdnf/vars /etc/dnf/vars /etc/yum/vars"
```

Comment:
can we make this a comma separated list?

Reply:
No, lists are always space separated. Same as for `baseurl` and others.